src/openforms/authentication/contrib/digid_eherkenning_oidc/tests/README.md

```diff
@@ -0,0 +1,25 @@
+# README
+
+The integration tests for the OIDC flavour of DigiD/eHerkenning uses VCR.py. To (re-) record the
+cassettes, perform the following steps (from the root of the repo):
```

Comment:
Do you need to set any VCR envvars to make this happen? Or is this not needed because you delete the existing cassettes?

Reply:
Deleting them is the easiest approach. There are envvars, but I find they don't behave as expected. Most notably, they will just append new requests to existing cassettes rather than replace them.

Deleting the files and then re-recording results effectively in "updates", and the benefit is that you get nice diffs of what has changed (should typically only be `Date` headers, etc)